cognee-mcp/Dockerfile

```
@@ -0,0 +1,49 @@
+ENV UV_LINK_MODE=copy
+
+
+# Install the project's dependencies using the lockfile and settings
+RUN --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,source=uv.lock,target=uv.lock \
+    --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
+    uv sync --frozen --no-install-project --no-dev --no-editable
+
+# Then, add the rest of the project source code and install it
+# Installing separately from its dependencies allows optimal layer caching
+ADD . /app
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)